src/software/jetson_nano/services/network/network.h

```diff
@@ -79,4 +102,12 @@ class NetworkService
 
     // track last breakbeam state for sending RobotStatus outside of specified rate
     bool last_breakbeam_state_sent = false;
+
+    /**
+     * Deque of key primitive set information as tuples for round-trip time calculations. Tuples specification:
+     * [0] uint64_t: Primitive Sequence Number
+     * [1] double: System time for when primitive set was received by Thunderloop in seconds
+     * [2] double: Epoch time of primitive set sent time from Thunderscope in seconds
```

Comment:
why do we need to store [2]?

Reply:
[2] is the time sent of the primitive set as you mentioned previously, I realized that this differs from the time sent within the `RobotStatus` proto as every primitive set will have a different time sent.